packages/dropdownable/tests/__snapshots__/Dropdownable.test.tsx.snap

```diff
@@ -259,19 +259,9 @@ exports[`Dropdownable is visible after opening 2`] = `
         className="emotion-1"
         overlayRoot={
           <body>
-            .emotion-0 {
-  top: 0px;
-  left: 0px;
-  position: absolute;
-  -webkit-transform: scale(1);
-  -ms-transform: scale(1);
-  transform: scale(1);
-  opacity: 1;
-}
-
-<div>
+            <div>
```

Comment:
I dont understand why this changed?  This looks like an unrelated class?

Reply:
I was surprised at first, but if styled was re-creating a component every render, it could be that it creates these styles each render as well, which was happening multiple times in this test. Now that the component is only created once, there's less style code created?

Reply:
I'm not too familiar with what emotion does under the hood

Reply:
@mperrotti we might need you to weigh in on this

Reply:
@brandonc's assessment sounds correct to me